docs/Community/governance/growth-plan.md

```diff
@@ -0,0 +1,51 @@
+# Growth Plan
+
+## Introduction
+
+The Open Enclave SDK is an
+[Incubation stage](https://github.com/confidential-computing/governance/blob/master/project-progression-policy.md#incubation-stage)
+project of the Confidential Computing Consortium (CCC), with an approved
+[Technical Charter](/TECHNICAL_CHARTER.md).
+
+One requirement for Incubation stage projects is to develop, in
+conjunction with the mentor assigned by the CCC Technical Advisory
+Council, a growth plan for reaching the
+[Graduation stage](https://github.com/confidential-computing/governance/blob/master/project-progression-policy.md#graduation-stage)
+over time.
+
+## Graduation Stage Acceptance Criteria
+
+This section tracks the progress towards the Acceptance Criteria for the
+Graduation Stage, as defined by the CCC:
+
+- [ ] Have a defined [governing body](README.md) of at least 5 or more members
+(owners and core maintainers, or similar technical role), of which no
+more than 1/3 is affiliated with the same employer. In the case there
+are 5 governing members, 2 may be from the same employer.
+
+- [ ] Have a healthy number of [committers](../Committers.md#list-of-committers) from at least two organizations.
+A committer is defined as someone with the commit bit; i.e., someone who
+can accept contributions to some or all of the project.
+
+- [X] Adopt a [Code of Conduct](../../CODE_OF_CONDUCT.md).
+
+- [X] Explicitly define a project [governance and committer process](../Governance.md). This
+is preferably laid out in a GOVERNANCE.md file and references a
+CONTRIBUTING.md and OWNERS.md file showing the current and emeritus
+committers.
+
+- [ ] Have a public list of project adopters or users for at least the
+primary repo (e.g., ADOPTERS.md or logos on the project website).
+
+- [ ] Other metrics as defined by the applying Project during the application
```

Comment:
Given that we're past the application process, how do we make progress on this item? Is it done? Is it something that needs to be included in the Plan below?

Reply:
My opinion is that we can mark it N/A

Reply:
@CodeMonkeyLeet I've now updated it to mark as N/A